roles/scm/github.com/tasks/main.yml

```diff
@@ -57,14 +57,15 @@
     read_only: "{{ item.deploy_key_read_only | default('no') }}"
     username: "{{ github_api_username }}"
     password: "{{ github_api_password }}"
-    otp: 123456
+    otp: "{{ otp | default('123456') }}"
```

Comment:
To align with the above username/password variables, can we name this variable `github_api_otp`?